### PR TITLE
Release 0.8

### PR DESCRIPTION
**Pull request by JanMarvin**
## prepare
* [x] `roxygen2::roxygenize()`
* [x] `spelling::update_wordlist()`
* [x] `spelling::spell_check_package()`
* [x] `lintr::lint_package()`
* [x] `R CMD build openxlsx2`
* [x] `R CMD check --as-cran`
* [x] run revdep
* [x] update NEWS

## submission
* "CRAN submissions will be offline from Jul 21, 2023 to Aug 7, 2023 (CRAN team vacation and maintenance work)."
* 2023-08-07 first CRAN submission

### CODE REVIEW

**Comment by JanMarvin** [comment]
Everything is ready for release `0.8`, but CRAN team is on vacation.

**Comment by JanMarvin** [comment]
rebased to include the last fix. should not change anything else

**Review by olivroy — COMMENTED** [review]
I think the force-merge missed that.

File: R/openxlsx2-package.R [hunk 1]
```diff
@@ -27,8 +27,7 @@
 #' * [wb_add_data_table()]
 #' * [wb_to_df()]
 #' * [read_xlsx()]
-#' * [write_xlsx()]
-#' * <https://janmarvin.github.io/openxlsx2>
+#' * <https://janmarvin.github.io/openxlsx2/>
```
[the remark this answers is not in the document]
Comment (JanMarvin) [reply R/openxlsx2-package.R:30]: ah, I missed that. rebased complained and I was guessing. Thanks again for noticing

**Pull request merged**